server/sonar-server/src/main/java/org/sonar/server/issue/IssueQueryService.java

```diff
@@ -223,11 +218,18 @@ public IssueQuery createFromRequest(SearchWsRequest request) {
       }
       return builder.build();
 
-    } finally {
-      session.close();
     }
   }
 
+  @CheckForNull
+  private String convertOrganizationKeyToUuid(DbSession dbSession, @Nullable String organizationKey) {
+    if (organizationKey == null) {
+      return null;
+    }
+    Optional<OrganizationDto> organization = dbClient.organizationDao().selectByKey(dbSession, organizationKey);
+    return organization.map(OrganizationDto::getUuid).orElse(UNKNOWN);
```

Comment:
Good idea but I won't do that right now. That should be applied to the 25 other fields which also can be optional.